test/lumo/test_util.cljs

```diff
@@ -0,0 +1,33 @@
+(ns lumo.test-util
+  (:require [cljs.nodejs :as node]
+            [cljs.js :as cljs]
+            [lumo.common :as common])
+  (:require-macros [cljs.env.macros :as env]))
+
+(defn with-parinfer [f]
+  (set! (. js/global -$$LUMO_GLOBALS) #js {:getParinfer #(node/require "parinfer")})
+  (f)
+  (set! (. js/global -$$LUMO_GLOBALS) nil))
+
+(def fs (js/require "fs"))
+
+(defn read-file-sync [file-path & [encoding-or-opts]]
+  (try
+    (.readFileSync fs file-path encoding-or-opts)
+    (catch :default e nil)))
+
+(def cljs-core-cache-path "generated_test_suite.out/cljs/core.cljs.cache.json")
+(def cljs-core-macros-cache-path "generated_test_suite.out/cljs/core$macros.cljc.cache.json")
```

Comment:
not a big deal, but this is relying on implementation details of `boot-cljs-test` which is subject to change. Can we add an explicit `ids` option to the `test` task in `build.boot`? Something like `:ids #{"test_out"}`